datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/UserController.java

```diff
@@ -38,28 +42,39 @@
      * public-keys: ${systemRoot}/${userName}/public/keystore
      */
     @PutMapping
-    @ApiOperation("Create user")
-    public void createUser(@RequestBody UserDTO requestDTO) {
-        ReadKeyPassword readKeyPassword = new ReadKeyPassword(requestDTO.getPassword());
-        UserIDAuth auth = new UserIDAuth(new UserID(requestDTO.getUserName()), readKeyPassword);
+    @ApiOperation("Creates new user")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "User successfully created"),
+            @ApiResponse(code = 400, message = "User already exists"),
+            @ApiResponse(code = 403, message = "Access denied")
+    })
+    public void createUser(@RequestBody UserDTO userDTO) {
+        ReadKeyPassword readKeyPassword = new ReadKeyPassword(userDTO.getPassword());
+        UserIDAuth auth = new UserIDAuth(new UserID(userDTO.getUserName()), readKeyPassword);
         if (dataSafeService.userProfile().userExists(auth.getUserID())) {
-            throw new UserExistsException("user \"" + auth.getUserID().getValue() + "\" already exists");
+            throw new UserExistsException("user '" + auth.getUserID().getValue() + "' already exists");
         }
         dataSafeService.userProfile().registerUsingDefaults(auth);
     }
 
     /**
      * Removes user.
-     * @param user username
+     *
+     * @param user     username
      * @param password user password.
      */
     @DeleteMapping
-    @ApiOperation("Delete user")
+    @ApiOperation("Deletes existing user")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "User successfully deleted"),
+            @ApiResponse(code = 403, message = "Access denied"),
```

Comment:
403 has default message, not necessary to add it

Reply:
removed